qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -36,12 +36,23 @@ class TwirlingOptions:
          those measurement registers not involved within a conditional logic.
          Default: True for Estimator, false for Sampler.
 
-        num_randomizations: The number of random samples to use when twirling or
-            peforming sampled mitigation. If "auto", the value will be chosen automatically
-            based on the input PUBs. Default: "auto".
+        num_randomizations: The number of random samples to use when twirling or peforming sampled mitigation.
+          If `num_randomizations` is "auto":
+
+          * If `shots_per_randomization` is not "auto", the value is set to `ceil(shots/shots_per_randomization)`,
+            where `shots` is the total number of shots per circuit and `ceil` is the ceiling function.
```

Comment:
In the sampler and estimator, the number of shots per circuit need not be uniform, which makes the phrase "total number of shots per circuit" possibly confusing. I would try to word it in terms of the shots of a pub.

Reply:
I gave my best shot at this, but open to better suggestions :)